shap/explainers/_tree.py

```diff
@@ -89,16 +125,16 @@ class TreeExplainer(Explainer):
     """
 
     def __init__(
-        self,
-        model,
-        data=None,
-        model_output="raw",
-        feature_perturbation="auto",
-        feature_names=None,
-        approximate=DEPRECATED_APPROX,
-        # FIXME: The `link` and `linearize_link` arguments are ignored. GH #3513
-        link=None,
-        linearize_link=None,
+            self,
```

Comment:
is there a particular reason for the formatting changes? I would to limit changes to either formatting or adding functionality, and not mix both into one PR.

Reply:
Hmm, I didn't change formatting although it is showed in the merge commit https://github.com/shap/shap/pull/3273/commits/cc326cc3ecb9d655443af66ca69c0b1ab5701066. Anyway the latest diff shows that nothing changed.